### DISCUSSION TITLE
How to quickly apply block format of a To-do list?

### DISCUSSION THREAD

**Discussion by WPum** [TriliumNext/Trilium#1600] [Q&A] [answered]
CKEditor 5 documentation says:
> Bulleted list – Start a line with * or - followed by a space.
Numbered list – Start a line with 1. or 1) followed by a space.
**To-do list – Start a line with [ ] followed by a space.**

I've tried it in Trilium but it didn't work. Is there any way to do that?

**Comment by jkaplon** [comment, marked as answer]
That's a new feature in CKEditor version 25.0.0 (recently released on 2021-01-27), [issue 7518](https://github.com/ckeditor/ckeditor5/issues/7518).
Trilium will get that version eventually, I'm looking forward to using it myself.
It **feels** like we might be getting a new Trilium release any day now, maybe it'll include new CKE hotness :).
Or mash that Fork button and get your hands dirty.

**Reply by zadam** [reply]
Good catch. CKEditor 25 is currently blocked by math support not being yet ported. New Trilium release is near but needs some testing still ...
